babel.config.js

```diff
@@ -34,6 +34,26 @@ const metro = {
     ],
 };
 
+const {CAPTURE_METRICS} = require('dotenv').config().parsed;
+
+/* When the CAPTURE_METRICS env var is set we add these aliases to also capture
+ * React.Profiler metrics for release builds */
+if (CAPTURE_METRICS) {
+    const path = require('path');
+    const profilingRenderer = path.resolve(
+        __dirname,
+        './node_modules/react-native/Libraries/Renderer/implementations/ReactNativeRenderer-profiling',
+    );
+
+    metro.plugins.push(['module-resolver', {
+        root: ['./'],
+        alias: {
+            'ReactNativeRenderer-prod': profilingRenderer,
+            'scheduler/tracing': 'scheduler/tracing-profiling',
+        },
```

Comment:
By default the Profiler component will do nothing on a release build 
This change allows to capture Performance stats for prod/release builds as well

This can be handy as we can also hook it to a Performance tracing backend similar to: https://itnext.io/react-native-profiler-43d131130c5c